src/main/scala/com/wavesplatform/settings/BlockchainSettings.scala

```diff
@@ -75,7 +75,8 @@ case class GenesisSettings(
   signature: Option[ByteStr],
   transactions: Seq[GenesisTransactionSettings],
   initialBaseTarget: Long,
-  averageBlockDelay: FiniteDuration)
+  averageBlockDelay: FiniteDuration,
+  featureCheckBlocksPeriod: Long)
```

Comment:
@alexeykiselev @phearnot 
I don't think this belongs to `GenesisSettings` at all, how about `FunctionalitySettings`?
Besides, isn't it `FeatureApprovalBlocksCount`?
If yes, why `MinBlocksCountToActivateFeature` isn't parameterized as well?
(relates to https://github.com/wavesplatform/Waves/pull/497/files)